### DISCUSSION TITLE
Converted 3 books now will not start

### DISCUSSION THREAD

**Discussion by markbharris** [yermak/AudioBookConverter#577] [General]
Hello, I am not sure what the issue is.  The source has not changed, they were all downloaded libby books.  Like it says converted 3 books now will not do anything.  Please help, I even bought the steam verison, which btw is a little weird.

**Comment by markmccoid** [comment]
Hi, I started having the same problem, although I'm using the portable version.

I went to "System" menu, then "Repair".  The app closed down.  Then I reopened and it started working.

Hope this helps.
